src/components/accordion/Accordion.spec.tsx

```diff
@@ -226,7 +226,14 @@ describe('<Accordion>', () => {
 
     expect(item.getAttribute('aria-expanded')).toEqual('false');
     expect(accordion.queryByRole('region')).toBeNull();
+
+    // This one does not trigger onFocus method in React 17
     accordion.getByRole('button').focus();
+
+    // This one does not set document.activeElement
+    // https://github.com/jsdom/jsdom/issues/2586
+    fireEvent.focus(accordion.getByRole('button'));
```

Comment:
I would suggest to move this into helper function (put it in the end of this file), use it here as 1 line and describe all the comments in that function. e.g.
```
focusElement(accordion.getByRole('button'));
```

and later

```js
function focusElement(element) {
    // we need both those calls for now due to defect in fireEvent.focus https://github.com/jsdom/jsdom/issues/2586
    // This alone does not trigger onFocus method
    element.focus(); 
    // This alone does not set document.activeElement due to bug
    fireEvent.focus(element);
}
```

This way it will be elegant in place of usage (and easy to replace later once defect fixed). And explanatory in the function if someone would like to dig deeper

Reply:
Fixed in https://github.com/brainly/style-guide/pull/2806/commits/768d6977da3d29ffcdaefd0cd91b83dd0b0d7c0c